news/_posts/core-weekly/2019-03-13-coreweekly-week-10-2019.md

```diff
@@ -0,0 +1,117 @@
+* [#12784](https://github.com/PrestaShop/PrestaShop/pull/12784): Add module main class parse error to log message. Thank you [@teemumantynen](https://github.com/teemumantynen)
+* [#12787](https://github.com/PrestaShop/PrestaShop/pull/12787): adds legacy links to all meta page routes. Thank you [@tomas862](https://github.com/tomas862)
+* [#12838](https://github.com/PrestaShop/PrestaShop/pull/12838): Update ProductSpecificPrice.php. Thank you [@metineren](https://github.com/metineren)
+
+
+### Front office
+
+* [#12748](https://github.com/PrestaShop/PrestaShop/pull/12748): Remove obsolete P3P policy. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#12753](https://github.com/PrestaShop/PrestaShop/pull/12753): Allow ipv6 to be used in htaccess redirects, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Installer
+
+* [#12666](https://github.com/PrestaShop/PrestaShop/pull/12666): Delivery slips fails when upgrading to 1.7.5. Thank you [@rblaurin](https://github.com/rblaurin)
+
+
+### Web services
+
+* [#12836](https://github.com/PrestaShop/PrestaShop/pull/12836): Fixes http 500 when trying to get the images of a product with no images . Thank you [@mowcixo](https://github.com/mowcixo)
+
+
+### Tests
+
+* [#12793](https://github.com/PrestaShop/PrestaShop/pull/12793): Allow Travis phpHigh build to fail as it is an informative check, not…, by [@matks](https://github.com/matks)
+* [#12804](https://github.com/PrestaShop/PrestaShop/pull/12804): Use string value for CSRF token in DeliveryControllerTest, by [@matks](https://github.com/matks)
+
+
+## Code changes in modules, themes & tools
+
+### Circuit breaker
+
+* [#26](https://github.com/PrestaShop/circuit-breaker/pull/26): Ease configuration of calls using all usual HTTP methods, by [@mickaelandrieu](https://github.com/mickaelandrieu)
+
+
+### Auto upgrade
+
+* [#285](https://github.com/PrestaShop/autoupgrade/pull/285): Use raw trans to avoid &quot; being displayed, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#286](https://github.com/PrestaShop/autoupgrade/pull/286): Allow ipv6 to pass the maintenance check, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#288](https://github.com/PrestaShop/autoupgrade/pull/288): Do not reset theme anymore when already active (prevent hooks to be reset), by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#290](https://github.com/PrestaShop/autoupgrade/pull/290): Do not display exception when running unit tests, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+## Changes in Documentation
+
+* [#224](https://github.com/PrestaShop/docs/pull/224): Add documentation on legacy controllers, by [@eternoendless](https://github.com/eternoendless)
+
+
+<hr />
+
+Thank you to the contributors whose pull requests were merged since the last Core Weekly Report: @mowcixo, @rblaurin, @sarjon, @tomas862, @toslan, @zuk3975!
```

Comment:
Some of them are from Invertus, do we still count them separately?

Reply:
They have always been listed in the contributors thanks.
But the real way would be to count everybody including the PrestaShop employees.